mbq/atomiq/apps.py

```diff
@@ -38,3 +38,4 @@ def ready(self):
         self.module.sqs_publish = sqs_producer.publish
         celery_producer = producers.CeleryProducer()
         self.module.celery_publish = celery_producer.publish
+        self.module.test_utils = test_utils
```

Comment:
why do we need this? I think imports would work without it no?

Reply:
yeah I still don't totally understand when we do/don't need to use the app module thing. I guess I can just remove it?

Reply:
I feel like we should either put this in an `__init__` or in `app.py` to make it obvious that is something that is part of atomiq's public interface. Since we're already using `app.py` we might as well keep using it?`

Reply:
Disagree. Use neither `__init__.py` nor `ready()`. This is only used in tests and it's perfectly reasonable to access it as `from mbq import atomiq.test` (or similar).

Reply:
okydoky

Reply:
i don't think we need that at all. we needed to be able to have the publish api without having to instantiate a producer class all the time.